packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/DownloadListenerHostApiImpl.java

```diff
@@ -5,40 +5,63 @@
 package io.flutter.plugins.webviewflutter;
 
 import android.webkit.DownloadListener;
-import io.flutter.plugins.webviewflutter.GeneratedAndroidWebView.DownloadListenerFlutterApi;
 
 class DownloadListenerHostApiImpl implements GeneratedAndroidWebView.DownloadListenerHostApi {
```

Comment:
Please make sure all public classes have comments, per style.

Reply:
Done. All pigeon api impl should be fully documented now